File: public/agendamento/model/Agendamento.php
<?php 
    class Agendamento {
        
        private $id;
        private $nome;
        private $dataConsulta;
        private $cpf;
        private $tipoExame;

        public function getId() {
            return $this->id;
        }

        public function setId($id) {
            return $this->id = $id;
        }

        public function getNome() {
            return $this->nome;
        }

        public function setNome($nome) {
            return $this->nome = $nome;
        }

        public function getDataConsulta() {
            return $this->dataConsulta;
        }

        public function setDataConsulta($dataConsulta) {
            return $this->dataConsulta = $dataConsulta;
        }

        public function getCpf() {
            return $this->cpf;
        }

        public function setCpf($cpf) {
            return $this->cpf = $cpf;
        }

        public function getTipoExame() {
            return $this->tipoExame;
        }

        public function setTipoExame($tipoExame) {
            return $this->tipoExame = $tipoExame;
        }

    }
?>

File: modelo/src/agendamento/model/Agendamento.java
package model;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class Agendamento {
    private int id;
    private String nome;
    private LocalDateTime dataConsulta;
    private String tipoExame;

    public Agendamento() {
    }

    public Agendamento(int id, String nome, LocalDateTime dataConsulta, String tipoExame) {
        this.id = id;
        this.nome = nome;
        this.dataConsulta = dataConsulta;
        this.tipoExame = tipoExame;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public LocalDateTime getDataConsulta() {
        return dataConsulta;
    }

    public void setDataConsulta(LocalDateTime dataConsulta) {
        this.dataConsulta = dataConsulta;
    }

    public String getTipoExame() {
        return tipoExame;
    }

    public void setTipoExame(String tipoExame) {
        this.tipoExame = tipoExame;
    }

    @Override
    public String toString() {
        return "Agendamento{" +
                "id=" + id +
                ", nome='" + nome + '\'' +
                ", dataConsulta=" + dataConsulta.format(DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm")) +
                ", tipoExame='" + tipoExame + '\'' +
                '}';
    }
}
